cookbook/pipelines/notebooks/README.md

```diff
@@ -4,6 +4,6 @@
 This directory contains several notebook-based examples of how to deploy a [Kubeflow Pipeline](https://www.kubeflow.org/docs/pipelines/) from outside the Kubeflow cluster.
 
 - [kfp_remote_deploy.ipynb](./kfp_remote_deploy.ipynb) shows how to do this with an IAP-enabled Kubeflow installation (on GKE). 
-- [kfp_remote_deploy-no-IAP.ipynb](./kfp_remote_deploy-no-IAP.ipynb) walks through how to connect via port-forwarding to the cluster
+- [kfp_remote_deploy-no-IAP.ipynb](./kfp_remote_deploy-no-IAP.ipynb) walks through how to deploy via a port-forwarding connection to a cluster.
```

Comment:
Does this example need to be updated now for 0.6? What method of deploying Kubeflow are you using for portforwarding?